contracts/permissionless/PermissionlessOrderBookReserveLister.sol

```diff
@@ -24,6 +37,10 @@ contract PermissionlessOrderBookReserveLister {
     mapping(address => ListingStage) public reserveListingStage;
     mapping(address => OrderBookReserveInterface) public reserves;
 
+    // KNC burn fee per order that is taken. = 25 / 1000 = 0.25 %
+    // TODO: pass this to OrderBookReserve's ctor
```

Comment:
why don't u pass it in Ctor?
u can do it!

Reply:
Done